ocaml/libs/ezxenstore/core/dune

```diff
@@ -9,5 +9,6 @@
     (re_export xenstore)
     (re_export xenstore_transport)
     threads.posix
+    xapi-stdext-threads
```

Comment:
There's a TODO in this commit, remove it, explaining the fix. I don't it should have a todo regarding upstreaming

Reply:
https://github.com/xapi-project/ezxenstore/pull/19